files-community/Files
# RichCommand: PermanentlyDelete

**Resolved / Related Issues**
   Related #11481

**Notes**
This PR also removes some `KeyboardAccelerators` leftovers (`Refresh` and `Delete`)

**Validation**
How did you test these changes?
- [] Did you remove any strings from the en-us resource file?
   1. Open app and select a file
   2. Use `Shift + Del` to permanently delete it or `Ctrl + D` to delete it
